examples/http-counter/src/lib.rs

```diff
@@ -1,52 +1,71 @@
 use anyhow::{Context, Result};
-use wasmcloud_component::http::{self, StatusCode};
```

Comment:
Had an annoying time with wit conflicts in wasm_component crate that I yanked it and am just using direct bindings.